install_config/imagestreams_templates.adoc

```diff
@@ -0,0 +1,321 @@
+
+ifdef::openshift-enterprise[]
+== Creating Image Streams for xPaaS Middleware Images
+The xPaaS Middleware image streams provide images for
+link:../using_images/xpaas_images/eap.html[*JBoss EAP*],
+link:../using_images/xpaas_images/jws.html[*JBoss JWS*],
+link:../using_images/xpaas_images/a_mq.html[*JBoss A-MQ*],
+link:../using_images/xpaas_images/fuse.html[*JBoss Fuse Integration Services*],
+link:../using_images/xpaas_images/decision_server.html[*Decision Server*], and
+link:../using_images/xpaas_images/data_grid.html[*JBoss Data Grid*]. They can be
+used to build applications for those platforms using the provided templates.
+
+To create the xPaaS Middleware set of image streams:
+
+----
+$ oc create -f $XPAASSTREAMDIR/jboss-image-streams.json -n openshift
+----
+
+[NOTE]
+====
+Access to the images referenced by these image streams requires the relevant
+xPaaS Middleware subscriptions.
+====
+endif::[]
+
+[[creating-database-service-templates]]
+
+== Creating Database Service Templates
+The database service templates make it easy to run a database image which can be
+utilized by other components. For each database
+(link:../using_images/db_images/mongodb.html[*MongoDB*],
+link:../using_images/db_images/mysql.html[*MySQL*], and
+link:../using_images/db_images/postgresql.html[*PostgreSQL*]),
+two templates are defined.
+
+One template uses ephemeral storage in the container which means data stored
+will be lost if the container is restarted, for example if the pod moves. This
+template should be used for demonstration purposes only.
+
+The other template defines a persistent volume for storage, however it requires
+your OpenShift installation to have
+link:../install_config/persistent_storage/persistent_storage_nfs.html[persistent volumes]
+configured.
+
+To create the core set of database templates:
+
+----
+$ oc create -f $DBTEMPLATES -n openshift
+----
+
+After creating the templates, users are able to easily instantiate the various
+templates, giving them quick access to a database deployment.
+
+[[creating-instantapp-templates]]
+
+== Creating InstantApp Templates
+The InstantApp templates define a full set of objects for a running application.
+These include:
+
+- link:../architecture/core_concepts/builds_and_image_streams.html#builds[Build configurations]
+to build the application from source located in a GitHub public repository
+- link:../architecture/core_concepts/deployments.html#deployments-and-deployment-configurations[Deployment configurations]
+to deploy the application image after it is built.
+- link:../architecture/core_concepts/pods_and_services.html#services[Services]
+to provide load balancing for the application
+link:../architecture/core_concepts/pods_and_services.html#pods[pods].
+- link:../architecture/core_concepts/routes.html[Routes]
+to provide external access to the application.
+
+Some of the templates also define a database deployment and service so the
+application can perform database operations.
```

Comment:
@bparees So are you suggesting the title of the section should be "Creating Quickstart Templates" instead? and remove references to InstantApp?